contracts/LaunchpegLens.sol

```diff
@@ -187,7 +207,7 @@ contract LaunchpegLens {
         ILaunchpegFactory factory = (_version == LaunchpegVersion.V1)
             ? launchpegFactoryV1
             : launchpegFactoryV2;
-        // 0 - Launchpeg, 1 - FlatLaunchpeg
+        // 0 - Launchpeg, 1 - FlatLaunchpeg, 2 - ERC1155SingleBundle
         uint256 lpTypeIdx = uint8(_type) - 1;
         uint256 numLaunchpegs = factory.numLaunchpegs(lpTypeIdx);
 
```

Comment:
can we add try/catch on getLaunchpegData? JourneyStamps have this hack https://github.com/traderjoe-xyz/journey-stamps/blob/67539b42c84d694e61afae63fb06b0b1a49d8918/contracts/JoeJourneyStamp.sol#L63 to return 0 on not-existing functions, but only when LaunchpegLens address is set. This call reverts, when another Lens are used

Reply:
I can't add a try catch on an internal function, I would need to do it for each individual calls to the contract... Maybe we can update the lens address on the two stamps contracts ?